src/settings/settings.vcxproj

```diff
@@ -60,6 +60,7 @@
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
       <AdditionalIncludeDirectories>..;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalOptions>/Zm150 %(AdditionalOptions)</AdditionalOptions>
```

Comment:
Does this improve the compile time?

Reply:
@enricogior It might be possible, but the main reason I've added it because of this compile-time error:
```
compiler limit: internal heap limit reached
```
This switch fixes them for me. Once the project grows larger we might need to increase this limit again.